package.json

```diff
@@ -105,7 +105,6 @@
     "@types/node": "12.12.27",
     "@types/react": "16.9.19",
     "@types/request": "2.48.4",
-    "@types/tus-js-client": "1.8.0",
```

Comment:
tus-js-client v2.1.0+ now includes the type definitions on its own, so that dependency is not needed anymore.